src/screens/Airdrop/ThankYouScreen.tsx

```diff
@@ -0,0 +1,96 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { Component } from 'react';
+import { Text, StyleSheet, View } from 'react-native';
+import { connect } from 'react-redux';
+
+import { ScreenTemplate, Button, Header } from 'app/components';
+import { Route } from 'app/consts';
+import { ApplicationState } from 'app/state';
+import { actions } from 'app/state/airdrop';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface MapStateProps {
+  thankYouSeen: boolean;
+}
+
+interface ActionProps {
+  markThankYouSeen: Function;
+}
+
+type Props = {
+  navigation: StackNavigationProp<any, Route.AirdropThankYou>;
+} & MapStateProps &
+  ActionProps;
+
+class AirdropThankYouScreen extends Component<Props, {}> {
+  componentDidMount() {
+    const { thankYouSeen, markThankYouSeen } = this.props;
+
+    if (!thankYouSeen) {
+      markThankYouSeen();
+    }
+  }
+
+  navigateBack = () => this.props.navigation.navigate(Route.Dashboard);
+
+  goToLearnMore = () => {
+    this.props.navigation.navigate(Route.AirdropRequirements);
+  };
+
+  render() {
+    return (
+      <ScreenTemplate
+        header={<Header isBackArrow title={i18n.airdrop.title} />}
+        footer={
+          <View style={styles.buttonsContainer}>
+            <Button
+              testID="cancel-button"
+              type="outline"
+              style={styles.iDontCareButton}
+              onPress={this.navigateBack}
+              title={i18n.airdrop.itIsAlive.iDontCare}
+            />
+            <Button
+              testID="learn-more"
+              onPress={this.goToLearnMore}
+              style={styles.learnMoreButton}
+              title={i18n.airdrop.itIsAlive.learnMore}
+            />
+          </View>
+        }
+      >
+        <Text style={styles.subtitle}>{i18n.airdrop.itIsAlive.subtitle}</Text>
+        <Text style={styles.description}>{i18n.airdrop.itIsAlive.description}</Text>
+      </ScreenTemplate>
+    );
+  }
+}
+
+const mapStateToProps = (state: ApplicationState): MapStateProps => ({
+  thankYouSeen: state.airdrop.thankYouSeen,
```

Comment:
use selectors

Reply:
yeah, I even had them implemented, but resigned. What is there reason to use sleectors here, in such a simple scenario? Am not converting store data, am not calculating anything, not changing form, just reaching out for simple value. Why use selectors here?

Reply:
https://github.com/reduxjs/reselect#reselect
`Selectors are efficient. A selector is not recomputed unless one of its arguments changes.`
Selectors are caching results so there are more performant and would be easier to reuse them in another selector if needed

Reply:
👍  thanks

Reply:
done